examples/branch/example/__init__.py

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Regarding this new directory in examples - I would rather just have the 2 existing examples changed to have the branch options on.